### DISCUSSION TITLE
Scrypted docker: Run as non-root user?

### DISCUSSION THREAD

**Discussion by iliyang** [koush/scrypted#1467] [Q&A] [answered]
I'm testing Scrypted NVR in a docker container on a Synology DS918+ NAS and it's been phenomenal so far. I only wonder if there's an option to make the container access the file system as a non-root user (for recordings & settings). Many docker images provide `PUID` and `PGID` environment variables to set the desired user and group id, to emulate a system user. Is this possible with the Scrypted container? Any guidelines?

**Comment by koush** [comment, marked as answer]
Currently docker requires root. However, the Proxmox container is not privileged.
